### DISCUSSION TITLE
How do you instegrate this with expo 50SDK?

### DISCUSSION THREAD

**Discussion by Jonatthu** [callstack/react-native-visionos#82] [Q&A]
Any help would be appreciated or hints!

**Comment by okwasniewski** [comment]
Hey @Jonatthu, 

Currently, it's not possible but I saw Expo adding support for React Native tvOS and React Native macOS in Expo SDK 50. So I think we are on a good track to support visionOS in the future. 

At this point, it's not our primary focus to add Expo support but I'm looking forward to it. Maybe you can also ask about it in the Expo repository under Discussions or Issues.
